pkg/yurtadm/cmd/cmd.go

```diff
@@ -45,7 +45,7 @@ func NewYurtadmCommand() *cobra.Command {
 	// add kubeconfig to persistent flags
 	cmds.PersistentFlags().String("kubeconfig", "", "The path to the kubeconfig file")
 	cmds.AddCommand(yurtinit.NewCmdInit())
-	cmds.AddCommand(join.NewCmdJoin(os.Stdout, nil))
+	cmds.AddCommand(join.NewCmdJoin())
```

Comment:
maybe we need to pass `os.Stdout` into `join.NewCmdJoin` in order to print log message prettily.

Reply:
I have set `os.Stdout` and `os.Stderr` in the `RunJoinNode`, does this already meet the requirements?
```go
// RunJoinNode executes the node join process.
func RunJoinNode(data joindata.YurtJoinData) error {
	if err := yurtadmutil.CheckAndInstallKubeadm(); err != nil {
		return err
	}

	kubeadmJoinConfigFilePath := filepath.Join(constants.KubeletWorkdir, constants.KubeadmJoinConfigFileName)
	kubeadmCmd := exec.Command("kubeadm", "join", fmt.Sprintf("--config=%s", kubeadmJoinConfigFilePath))
	kubeadmCmd.Stdout = os.Stdout
	kubeadmCmd.Stderr = os.Stderr

	if err := kubeadmCmd.Run(); err != nil {
		return err
	}

	return nil
}
```